packages/shopify-app-express/docs/reference/shopifyApp.md

```diff
@@ -8,7 +8,7 @@ This function creates an object that contains everything an Express app needs to
 
 `ApiConfigParams` | :exclamation: required when not using the Shopify CLI
 
-All values allowed by the `@shopify/shopify-api` package [when calling `shopifyApi`](https://github.com/Shopify/shopify-api-js/blob/shopify_api_next/README.md#configurations).
+All values allowed by the `@shopify/shopify-api` package [when calling `shopifyApi`](https://github.com/Shopify/shopify-api-js/blob/main/docs/reference/shopifyApi.md).
```

Comment:
We should add a pattern to the link check to fail if any link that has `blob` or `tree` does NOT have `main` after it.